lighthouse-core/lib/csp-evaluator.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @typedef Finding
+ * @property {number} type
+ * @property {string} description
+ * @property {number} severity
```

Comment:
number between what and what?

Reply:
The scale is 0-100 although lowest is 10:
https://github.com/google/csp-evaluator/blob/f40c18cbe35f3ac5a6c7d04da64f7078b9fca4b1/evaluator/finding.js#L58-L66